language/resource-api/README.md

```diff
@@ -465,42 +481,9 @@ To allow implementors a wide latitude in implementing connection and retry handl
 
 1. Any other methods on the transport are to be used by providers and tasks talking to this kind of target. Those operations are free to use any error signalling mechanism that is convenient. Providers and tasks will have to provide appropriate error signalling to the runtime.
 
-### Direct Access to Transports
-
-It is possible to use a RSAPI Transport directly using the `connect` method:
-
-`Puppet::ResourceApi::Transport.connect(name, config)`
-
-1. `register` the transport schema for the remote resource by:
-    * Directly calling into `Puppet::ResourceApi.register_transport`
-    * Loading an existing schema using `require 'puppet/transport/schema/<transportname>`
-    * Setting up Puppet's autoloader for `'puppet/transport/schema'`
-2. `connect` to the transport by name, passing the connection info
-3. When the transport has been initialized, `connect` will return the `Transport` object
-
-See above for possible exceptions coming from initializing a transport.
-
-To get a list of all registered transports, call the `list` method:
-
-`Puppet::ResourceApi::Transport.list`
-
-It will return a hash of all registered transport schemas keyed by their name. Each entry in the list is the transport schema definition as passed to `register_transport`:
+#### Bridging into Puppet
 
-
-```ruby
-{
-  'nexus' => {
-    name: 'nexus',
-    desc: 'Connects to a Cisco Nexus device',
-    # ...
-    # the rest of the transport schema definition here
-  },
-}
-```
-
-### Legacy Support
-
-Before Puppet 6.X (TBD), remote resources were only supported through the `Puppet::Util::NetworkDevice` namespace. To make a module useful on these older versions, a shim `Device` class needs to be provided to connect the dots:
+Before the Resource API, remote resources were only supported through the `Puppet::Util::NetworkDevice` namespace. To connect your transport to what Puppet expects, you need to provide a shim `Device` class.
```

Comment:
Maybe change: "To connect your transport to what Puppet expects, you need to provide a shim `Device` class."

To: "To connect your transport *in a way that* Puppet *understands*, you need to provide a shim `Device` class."

If this is what it means? If not please clarify.

Reply:
That sounds so much better.